### ISSUE TITLE
Build fails on windows

### ISSUE THREAD

**Issue by DenuxPlays** [ProbablyClem/utoipauto#10]
The following line works on Linux without any Problems but when compiled I get a few errors.
The line:
```rs
#[utoipauto(paths = "./backend/src")]
```
More context can be found here:
https://github.com/financrr/backend/blob/d883222278ea5217f0854c6a1b86ff900b71383b/backend/src/main.rs#L48

The error:
![image](https://github.com/ProbablyClem/utoipauto/assets/83671398/3a9e0f0a-2c71-4ec7-9025-bc63a247074c)

When I add a trailing `,` I get the error that the crate could not parse the file.

**Comment by DenuxPlays** [comment]
Specifying a full path to a file works though.

**Comment by DenuxPlays** [comment]
Okay.
I give up.
It simply does not work on windows. 
This fucking annoying....

**Comment by DenuxPlays** [comment]
Things I tried:
```rs
#[utoipauto(paths = "./backend/src")]
#[utoipauto(paths = ".\\backend\\src")]
#[utoipauto(paths = ".\\/backend\\/src")]
#[utoipauto(paths = "backend/src")]
#[utoipauto(paths = "/backend/src")]
#[utoipauto(paths = "./backend/src/")]
#[utoipauto(paths = "backend/src")]
```

I don't know what to do...
There is no alternative: wsl2 to slow when compiling, docker compiles everytime everything
Please make this work.
I'm frustrated.
I don't fucking know where the stupd ass compiler takes the fucking error `unknown start of token: \`  when only using `/`.

**Comment by ProbablyClem** [comment]
Hey, 
Weird issue, I've never had issues on windows (but I rarely use it).
I'll look into it

**Comment by DenuxPlays** [comment]
> Hey, Weird issue, I've never had issues on windows (but I rarely use it). I'll look into it

Yes.
After thinking about it (and take a good amount of sleep lol) I guess that he tries to replace the `/` with `\` because Windows doesn't uses `/` in their file system.
But that just a guess

**Comment by DenuxPlays** [comment]
Had you anytime to take a look at this?

**Comment by ProbablyClem** [comment]
Nope sorry

Le lun. 22 janv. 2024, 19:33, Timon Klinkert ***@***.***> a
écrit :

**Comment by ProbablyClem** [comment]
Hey, It's fixed on 0.1.6.
It was obviously a dump backslash error.
Thanks for your feedback

**Comment by DenuxPlays** [comment]
> Hey, It's fixed on 0.1.6.
> 
> It was obviously a dump backslash error.
> 
> Thanks for your feedback


Thank you
Gonna test that a little bit later
But really really thank you 🙏

**Comment by DenuxPlays** [comment]
Thank you
Works fine now :)
